src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1133,7 +1133,22 @@ private uint getClassSize(CORINFO_CLASS_STRUCT_* cls)
         }
 
         private uint getClassAlignmentRequirement(CORINFO_CLASS_STRUCT_* cls, bool fDoubleAlignHint)
-        { throw new NotImplementedException("getClassAlignmentRequirement"); }
+        {
+            if (_compilation.TypeSystemContext.Target.Architecture == TargetArchitecture.ARM ||
```

Comment:
I have the same opinion, I just test only on ARM. Therefore, I decided not to change the usual logic.
But if you think that this change need and can be extended to other architectures, I think it will be better.